cockatrice/src/gamesmodel.cpp

```diff
@@ -472,10 +461,10 @@ bool GamesProxyModel::filterAcceptsRow(int sourceRow) const
     if ((int)game.max_players() > maxPlayersFilterMax)
         return false;
 
-    if (maxGameAgeSeconds != DEFAULT_MAX_GAME_AGE_SECONDS) {
-        QDateTime then;
-        then.setTime_t(game.start_time());
-        if (then.secsTo(QDateTime::currentDateTime()) > maxGameAgeSeconds) {
+    if (maxGameAge.isValid()) {
```

Comment:
instead of checking if it's the default value, which not necessarily means it is nothing, we check if it's correctly formatted, an unset QTime() is invalid